action/protocol/execution/evm/evmstatedbadapter_test.go

```diff
@@ -319,7 +319,7 @@ func TestNonce(t *testing.T) {
 var tests = []stateDBTest{
 	{
 		[]bal{
-			{_addr1, big.NewInt(40000)},
+			{_addr1, uint256.NewInt(40000)},
```

Comment:
no need to change, just convert to uint256 when it is used

Reply:
reverted